examples/request_no_blocking_main_thread/pokemon_client.rs

```diff
@@ -0,0 +1,41 @@
+use serde::Deserialize;
```

Comment:
The extra file doesn't contain much, so I would just put everything in a single container file